woocommerce/woocommerce
# Analyzer: Turn on comments for database changes

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/platform-private/issues/51

### How to test the changes in this Pull Request:

1. Create a test branch from this one.
2. Add and commit a database update, by adding an array similar to this one
https://github.com/woocommerce/woocommerce/blob/e4824b9660e06dbcf65de3e46231275f9b606701/plugins/woocommerce/includes/class-wc-install.php#L209-L211
3. Modify and commit a schema change in any or all of these
https://github.com/woocommerce/woocommerce/blob/e4824b9660e06dbcf65de3e46231275f9b606701/plugins/woocommerce/includes/class-wc-install.php#L1014
https://github.com/woocommerce/woocommerce/blob/e4824b9660e06dbcf65de3e46231275f9b606701/plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php#L828
https://github.com/woocommerce/woocommerce/blob/e4824b9660e06dbcf65de3e46231275f9b606701/plugins/woocommerce/src/Internal/ProductAttributesLookup/DataRegenerator.php#L473
4. See resulting comment once Github Actions run
![image](https://user-images.githubusercontent.com/1922453/170628722-3ecb8773-a0c5-40fd-b845-d963b8008726.png)


### Other information:




### FOR PR REVIEWER ONLY:
